crates/compiler/alias_analysis/src/lib.rs

```diff
@@ -136,12 +136,15 @@ pub fn spec_program<'a, I>(
     arena: &'a Bump,
     interner: &STLayoutInterner<'a>,
     opt_level: OptLevel,
-    opt_entry_point: Option<roc_mono::ir::EntryPoint<'a>>,
+    entry_points: &'a [(Symbol, ProcLayout<'a>)],
     procs: I,
 ) -> Result<morphic_lib::Solutions>
 where
     I: Iterator<Item = &'a Proc<'a>>,
 {
+    // TODO support multiple entry points here!
+    let opt_entry_point = entry_points.first();
```

Comment:
We now have multiple host entrypoints propagated this far, but alias analysis hasn't been updated to support that concept yet. So, progress but not all the way to the payoff yet!